docs/getting-started.md

```diff
@@ -53,50 +55,61 @@ $ git log --show-signature
 Build and install globally:
 ```
 $ npm rebuild
-$ npm install -g # link globally
+$ npm install --global
 ```
 
-If you're updating a repository it is necessary to run `npm rebuild` again
-if any dependencies with native addons have been updated.
+Note: If you're updating a repository it is necessary to run `npm rebuild`
+again if any dependencies with native addons have been updated.
 
-### Installing on Debian/Ubuntu
+### Installing via GPK
 
-Install the necessary dependencies in addition to Node.js:
+To support signature verification, you can use [`gpk`][gpk] to replace the
+use of `npm`.
+
+To install `bcoin` globally and to your path:
 ```
-apt-get install build-essential python
+$ gpk install --global https://github.com/bcoin-org/bcoin
```

Comment:
I wonder if this should specify `...#master`
At this time, running this command pulls bcoin v1.0.2 (last release that was tagged?)
SO maybe this isn't an issue once v2.0.0 is officially tagged

Reply:
Yeah, not an issues once v2.0.0 is tagged, which this is in preparation.

Reply:
Do you think it should specify "To install _latest tagged release_ globally and add to your path" ?

(p.s. nit: ADD to your path)

Reply:
oh nm, it says latest version below. Maybe just insert the word 'add' here?